tests/storage/storage.py

```diff
@@ -196,6 +196,9 @@ def coalesce_integrity(vm: VM, vdi: VDI, vdi_op: CoalesceOperation):
 XVACompression = Literal['none', 'gzip', 'zstd']
 
 def xva_export_import(vm: VM, compression: XVACompression):
+    # The tests using this function are using specific fixtures to create the VM on the expected SR
```

Comment:
Is there any reason to not use docstring here ? maybe you don't want it to be considered like a public api ?